tests/unit/registry-helper-test.js

```diff
@@ -0,0 +1,48 @@
+import Ember from 'ember'
+import hbs from 'htmlbars-inline-precompile'
+import {registryHelper} from 'ember-test-utils'
+import {describeComponent, it} from 'ember-mocha'
+import {expect} from 'chai'
+import resolver from '../helpers/resolver'
+
+const ListComponent = Ember.Component.extend({
+  classNames: ['.list']
```

Comment:
Seems odd to have the `.` in the class name here (and below) even if it is in a test...